### ISSUE TITLE
[A11y][AAA] WCAG No image of text

### ISSUE THREAD

**Issue by chunhtai** [flutter/flutter#170700]
### Use case

WCAG Images of Text (No Exception) (Level AAA) SC 1.4.9
https://www.w3.org/WAI/WCAG21/Understanding/images-of-text-no-exception

suggests there shouldn't be text in image unless they are pure decoration purposes

### Proposal

We should have a guideline that detects whether there is text in image. This would require OCR as a pure dart package which is currently not supported yet.

**Comment by chunhtai** [comment]
filed https://github.com/googlesamples/mlkit/issues/953
